.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  FORC_VERSION: 0.13.2
+  FORC_VERSION: 0.14.2
```

Comment:
It probably won't make any difference, but can this be bumped to `0.14.4`? If there's an incompatibility then it can be deferred to a later PR.